tests/Microsoft.Identity.Test.Common/TestConstants.cs

```diff
@@ -462,4 +462,15 @@ internal static class Adfs2019LabConstants
         public const string ADFS2019ClientSecretURL = "https://buildautomation.vault.azure.net/secrets/ADFS2019ClientCredSecret/";
         public const string ADFS2019ClientSecretName = "ADFS2019ClientCredSecret";
     }
+
+    //MSI Azure resources
+    public enum MsiAzureResource

```

Comment:
In the MSI PR, I have created a class to hold MSI related enums, lets place this enum there as well: https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/blob/nebharg%2FMsiApiPrototype/tests/Microsoft.Identity.Test.Common/Core/Helpers/ManagedIdentityTestUtil.cs#L10

Reply:
I will wait for you to merge and then take this change